### PR TITLE
Drinks are ordered in alphabetic order

### PR DESCRIPTION
**Pull request by tistatos**

### CODE REVIEW

**Comment by klaseskilson** [comment]
This only changes the order of the drink types though? Or do the drinks get ordered too?

**Comment by klaseskilson** [comment]
@tistatos question for you ☝️

**Comment by tistatos** [comment]
only drink types right now - will fix that

**Comment by danielronnkvist** [comment]
There's an easier way! If we set the model scope to be:
```ruby
scope :ordered_by_title, -> { order(name: :asc) }
```
Every time we get drinks they will be ordered by name.

**Comment by danielronnkvist** [comment]
Woops! Sorry, should be `default_scope`. Not just `scope`.

**Comment by tistatos** [comment]
I think we could :ship: :it:

**Comment by klaseskilson** [comment]
👀

File: app/controllers/drinks_controller.rb [hunk 1]
```diff
@@ -117,4 +117,3 @@ def drink_params
       params.require(:drink).permit(:name, :brewery, :country, :percentage, :price, :drink_type_id, :description, :instock, :label_url)
     end
 end
-
```
Comment (klaseskilson) [inline app/controllers/drinks_controller.rb]: 😻

File: app/models/drink_type.rb [hunk 2]
```diff
@@ -1,5 +1,8 @@
 class DrinkType < ActiveRecord::Base
   extend FriendlyId
   friendly_id :name, use: :slugged
-	has_many :drinks
+
+  default_scope { order(name: :asc) }
+
+  has_many :drinks
```
Comment (klaseskilson) [inline app/models/drink_type.rb]: 👌

**Comment by klaseskilson** [comment]
Are the changes to `Gemfile.lock` relevant? Or could we handle them separately?

**Comment by klaseskilson** [comment]
@tistatos ☝️

**Comment by tistatos** [comment]
Nope they aren't!

**Comment by klaseskilson** [comment]
So perhaps they can be removed from this PR in order to keep things separated?

**Comment by tistatos** [comment]
:eyes:

**Comment by klaseskilson** [comment]
Great! LGTM! 🆗 🚂

**Pull request merged**